src/main/webapp/data/userMapData.json

```diff
@@ -379,11 +388,14 @@
       "ISEN",
       "Lille 1 University",
       "Lille Catholic University",
+      "Paris 7",
       "Sciences Po",
       "SKEMA Business School",
       "Sup'Biotech",
       "Telecom Paris Tech",
-      "University of Lyon"
+      "University of Lyon",
+      "University of Savoy",
+      "Université Savoie Mont Blanc"
```

Comment:
I didn't find these two universities in the list... is it the university you mentioned above that has two names in different languages?

Reply:
This one has a bunch of alternate names...I could just switch to the one listed in the issue

https://en.wikipedia.org/wiki/University_of_Savoy

Reply:
I think "Université Savoie Mont Blanc" should be an official name according to that logo. We can keep this one and remove "University of Savoy".